### PR TITLE
BANES Confirm integration

### PR DESCRIPTION
**Pull request by davea**
This PR adds all of the BANES-specific code for their integration. I have left the commits mostly as they occurred so you can see the history of how things progressed and perhaps the rationale behind the structure.

### CODE REVIEW

File: perllib/Integrations/Confirm/BANES.pm [hunk 1]
```diff
@@ -15,4 +15,15 @@ sub credentials {
     );
 }
 
+sub server_timezone { 'Europe/London' }
+
+has '+memcache_namespace'  => (
+    is => 'ro',
```
Comment (dracos) [inline perllib/Integrations/Confirm/BANES.pm]: No need for this line if you're using the `+`.
(Same applies to all the attributes in this PR.)

File: perllib/Open311/Endpoint/Integration/UK/BANES.pm [hunk 2]
```diff
@@ -5,52 +5,161 @@ use Moo;
 
 use Integrations::Confirm::BANES;
 
-
 has jurisdiction_id => (
     is => 'ro',
     default => 'banes_confirm',
 );
 
-sub integration_class { 'Integrations::Confirm::BANES' }
```
Comment (dracos) [inline perllib/Open311/Endpoint/Integration/UK/BANES.pm]: Guess would be good to not have all this in the parent PR if it's all now changed.

Comment (davea) [reply perllib/Open311/Endpoint/Integration/UK/BANES.pm]: Not sure I follow you here. `integration_class` comes back below as an attribute.

Comment (dracos) [reply perllib/Open311/Endpoint/Integration/UK/BANES.pm]: I just meant I don't really want any BANES-specific stuff in #6 especially if it's then rewritten in this PR :)

Comment (davea) [reply perllib/Open311/Endpoint/Integration/UK/BANES.pm]: Ah I see what you mean. I tried to break the PRs apart at the most logical point, but the commit history has the BANES-specific code fairly intertwined as that's how it progressed. I can revisit it and extract the council-specific stuff out of #6 if you want, but once #6, #7 and #8 are merged I don't think it'd be worth the hassle.

**Comment by dracos** [comment]
This and #8 now build upon config being in an external YAML file as approved in #12. I've kept copies of old branches so we can set up the YAML files in config (or if something is wrong with my rebasing).

**Comment by dracos** [comment]
The merge fails because it doesn't seem to take the change to .travis.yml along, not sure why not. But it should be fine.

**Pull request merged**